src/adapters/rhythmone.js

```diff
@@ -150,120 +159,77 @@ module.exports = function(bidManager, global, loader){
     });
 
     function p(k,v){
+      if(v instanceof Array)
+        v = v.join(",");
       if(typeof v !== "undefined")
         query.push(encodeURIComponent(k)+"="+encodeURIComponent(v));
     }
+
+    p("domain", attempt(function(){
+        var d = global.document.location.ancestorOrigins;
+        if(d && d.length > 0)
+          return d[d.length-1];
+        return global.top.document.location.hostname;
```

Comment:
The attempt function wraps this callback in a try/catch - somewhere around line 78